docs/faq.md

```diff
@@ -87,3 +88,613 @@ TBD
+              icon: "favorite-list",
+              viewUrl:
+                "https://fiddle.luigi-project.io/examples/microfrontends/multipurpose.html",
+              hideSideNav: true,
+              context: {
+                title: "Left navigation hidden",
+                content:
+                  "for pages needing more space or wanting to handle navigation internally",
+              },
+            },
+            {
+              pathSegment: "help",
+              label: "Help",
+              icon: "sys-help",
+              viewUrl:
+                "https://fiddle.luigi-project.io/examples/microfrontends/multipurpose.html",
+              context: {
+                title: "Help Section",
+                content: "Find some useful links on the left",
+              },
+              children: [
+                {
+                  label: "Back",
+                  link: "/",
+                  icon: "nav-back",
+                },
+                {
+                  label: "Luigi Github Page",
+                  externalLink: {
+                    url: "https://github.com/kyma-project/luigi",
+                  },
+                },
+                {
+                  label: "Fundamental Library",
+                  externalLink: {
+                    url: "https://sap.github.io/fundamental-styles",
+                  },
+                },
+                {
+                  label: "Fundamental Icons",
+                  externalLink: {
+                    url:
+                      "https://sap.github.io/fundamental-styles/components/icon.html",
+                  },
+                },
+              ],
+            },
+          ],
+        },
+​
+        routing: {
+          useHashRouting: true,
+        },
+        settings: {
+          hideNavigation: true /* IMPORTANT, hides luigi shell */,
+        },
+        lifecycleHooks: {
+          luigiAfterInit: () => {
+            init(); /* IMPORTANT, calls init from custom app logic */
+          },
+        },
+      });
+    </script>
+​
+    <!-- LUIGI CONFIG -->
+​
+    <!-- HTML -->
+​
+    <div class="myHeader">
+      <img src="https://fiddle.luigi-project.io/img/luigi.png" />
+      <span>ACME Corp</span>
+      <div class="links"></div>
+    </div>
+​
+    <div class="myLeftSideNav"></div>
+​
+    <div luigi-app-root></div>
+    <!-- IMPORTANT -->
+​
+    <!-- HTML -->
+  </body>
+</html>
+```
+
+### How do I disable the Luigi loading indicator? / My micro frontend page is stuck on a loading screen.
+
+If you include micro frontends in Luigi which don't use Luigi Client, Luigi Core won't know when they are ready because there is no handshake. This can cause the page to remain loading indefinitely. In that case, you should disable the loading indicator using the [loadingIndicator.enabled]https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=loadingindicatorenabled parameter in your configuration file.
```

Comment:
Something wrong in the link here, didn't appear there: 

```suggestion
If you include micro frontends in Luigi which don't use Luigi Client, Luigi Core won't know when they are ready because there is no handshake. This can cause the page to remain loading indefinitely. In that case, you should disable the loading indicator using the [loadingIndicator.enabled](https://docs.luigi-project.io/docs/navigation-parameters-reference/?section=loadingindicatorenabled) parameter in your configuration file.
```

Reply:
Other than this LGTM 👍